pcdsdevices/sqr1.py

```diff
@@ -0,0 +1,43 @@
+"""
+Ophyd class for square-one tri-sphere motion system (https://www.sqr-1.com/tsp.html)
+"""
+
+import logging
+
+from ophyd.device import Device
+from ophyd.device import FormattedComponent as FCpt
+from ophyd.pv_positioner import PVPositionerIsClose
+from ophyd.signal import EpicsSignal, EpicsSignalRO
+
+logger = logging.getLogger(__name__)
+
+
+class SQR1Axis(PVPositionerIsClose):
+    setpoint = FCpt(EpicsSignal, '{self.prefix}:TARGET:{self._axis}', kind='normal')
+    readback = FCpt(EpicsSignal, '{self.prefix}:TARGET:{self._axis}:RBV', kind='normal')
+    actuate = FCpt(EpicsSignal, '{self.prefix}:MOV', kind='normal')
+    actuate_value = 1
+    stop_signal = FCpt(EpicsSignal, '{self.prefix}:KILL', kind='normal')
+    stop_value = 1
+
+    llm = FCpt(EpicsSignalRO, '{self.prefix}:TARGET:{self._axis}:LLM', kind='normal')
+    hlm = FCpt(EpicsSignalRO, '{self.prefix}:TARGET:{self._axis}:HLM', kind='normal')
+
+    def __init__(self, axis, **kwargs):
+        self._axis = axis  # axis values {X,Y,Z,rX, rY, rZ}
+        super().__init__(**kwargs)
```

Comment:
This behaves a bit differently to all other devices because the positional `prefix` argument has been made keyword-only and `axis` is now the first argument instead of `prefix`. I recommend undoing this so that people don't get confused when trying to instantiate this device, and to make it easier to include this device as a component.

```suggestion
    def __init__(self, prefix, axis, **kwargs):
        self._axis = axis  # axis values {X,Y,Z,rX, rY, rZ}
        super().__init__(prefix, **kwargs)
```